Loop/Managers/AppStoreVersionCheckService.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  AppStoreVersionCheckService.swift
+//  Loop
+//
+//  Created by Rick Pasetto on 10/4/21.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+
+class AppStoreVersionCheckService: VersionCheckService {
```

Comment:
The term "AppStore" implies this is specific to app store delivery only. For DIY, there are a variety of app delivery mechanisms. Can we make this class more generic?  Like AppVersionCheckService?

Reply:
To me this is correct, as-is. This is an *App Store* version check service. I suppose to be even more clear it could be `AppleAppStoreVersionCheckService`. If DIY wants to implement a `VersionCheckService` for another app delivery mechanism, that's fine, but it would be a different implementation of `VersionCheckService`.